tools/clang/tools/dxcompiler/dxcapi.cpp

```diff
@@ -60,8 +59,9 @@ HRESULT CreateDxcContainerBuilder(REFIID riid, _Out_ LPVOID *ppv) {
   // Call dxil.dll's containerbuilder
   *ppv = nullptr;
   const char *warning;
-  HRESULT hr = DxilLibCreateInstance(CLSID_DxcContainerBuilder,
-                                     (IDxcContainerBuilder **)ppv);
+  HRESULT hr =
+      S_OK; // originally: DxilLibCreateInstance(CLSID_DxcContainerBuilder,
```

Comment:
We'll need to remove more of this code, basically skipping everything before where we create the internal container builder with `DxcContainerBuilder::Alloc`.  We also no longer need the warning, so we can remove the warning param from `DxcContainerBuilder::Init`, `DxcContainerBuilder::m_warning`, and all related code.